src/jslisten.c

```diff
@@ -537,28 +592,30 @@ int bindJoy(void) {
 
   // Non-blocking reading
   struct js_event js;
-  int needTrigger;
+  int triggerSection;
```

Comment:
renamed to call out this is referencing the `macro` to call ... which now as I write this `section` renamed as `macro` globally may be nice --